packages/dappmanager/test/testIpfsUtils.ts

```diff
@@ -56,3 +57,29 @@ export async function ipfsAddManifest(manifest: Manifest): Promise<string> {
   const addResult = await ipfsAdd(content);
   return addResult.hash;
 }
+
+/**
+ * Set Up an IPFS node for testing purposes on localhost.
+ * source: https://docs.ipfs.io/how-to/run-ipfs-inside-docker/
+ */
+export async function setUpIpfsNode(): Promise<void> {
+  await shell(
+    `docker run -d --name ${ipfsTestContainerName} -v ipfsdnpdappnodeeth_export:/export -v ipfsdnpdappnodeeth_data:/data/ipfs -p 4001:4001 -p 4001:4001/udp -p 127.0.0.1:8080:8080 -p 127.0.0.1:5001:5001 ipfs/go-ipfs:v0.9.1`
+  );
+  // Timeout for container to be initialized
+  await sleep(30000);
```

Comment:
Find a better faster strategy, like polling the API's version route every 1 second then resolving